cli/tests/testdata/test/junit.out

```diff
@@ -0,0 +1,118 @@
+Check file:///Users/alexanderrodionmichaud/code/deno/cli/tests/testdata/test/junit.ts
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="deno test" tests="16" failures="0" errors="0" time="0.136">
+    <testsuite name="file:///Users/alexanderrodionmichaud/code/deno/cli/tests/testdata/test/junit.ts" tests="16" disabled="0" errors="0" failures="0">
```

Comment:
@skycoop should we use file URLs here or strip the prefix?

Reply:
Oh good catch we should definitely not assert that it matches my filesystem 😅